routes/clucks/clucksRouter.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../../db/client');
+
+function timeDifference(current, previous) {
```

Comment:
replicated the function in `index`? never replicate a function that you can re-use.